python/cuda_parallel/tests/test_reduce.py

```diff
@@ -45,9 +45,9 @@ def op(a, b):
         num_items = 2**num_items_pow2
         h_input = random_int(num_items, dtype)
         d_input = numba.cuda.to_device(h_input)
-        temp_storage_size = reduce_into(None, d_input, d_output, None, h_init)
+        temp_storage_size = reduce_into(None, d_input, d_output, len(d_input), h_init)
```

Comment:
Do not have intuition for how expensive the call to `len(d_input)` is, but I would save it and reused the result few lines below.

Reply:
It's really quite fast, and in the context of these unit tests, storing the result in a variable wouldn't really save us much. In this case, I don't think it's necessary:

```python
In [4]: arr = cp.asarray([1, 2, 3])

In [5]: %timeit arr.size
16.7 ns ± 0.0128 ns per loop (mean ± std. dev. of 7 runs, 100,000,000 loops each)

In [6]: %timeit len(arr)
21.2 ns ± 0.866 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```